test/integration/test_kubernetes_staging.py

```diff
@@ -0,0 +1,117 @@
+"""Integration tests for Kubernetes pod staging.
+
+This case is way more brittle than the other integration tests distributed with Galaxy.
+This is because it requires Kubernetes, RabbitMQ, and the test itself needs to know what
```

Comment:
Probably makes some sense for the test to spin up a rabbitmq docker container ?

Reply:
@mvdbeek Indeed, I started down that road but I got worried about something... whether the Kubernetes containers would be able to talk to the MQ container maybe... but that seems odd because you'd probably bind its ports to a port on the host. It is a good idea and would make this more robust I think.

@VJalili did you make any progress on MQ tests for Galaxy/Pulsar? If you're still working on or planning to work on that task, it might be worth shifting focus to this context and make this change here as @mvdbeek suggests. I'm not sure if this is a better start than what I had previously pointed you at or if it just more moving pieces to get right.